__tests__/app.test.js

```diff
@@ -80,95 +81,98 @@ describe("GET /api/articles/:article_id", () => {
 })
 
 describe("GET /api/articles", () => {
-  test("200: Responds with an array of article objects", () => {
+  describe("GET /api/articles", () => {
+    test("200: Responds with an array of article objects", () => {
+          return request(app)
+          .get("/api/articles")
+          .expect(200)
+          .then(({body: { articles }}) => {
+            expect(articles).toHaveLength(13)
+            articles.forEach((article) => {
+              expect(article).toMatchObject({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(Number)
+            });
+          })
+        })
+      })
+
+    test("200: Article objects do not contain a body property", () => {
       return request(app)
       .get("/api/articles")
       .expect(200)
       .then(({body: { articles }}) => {
         expect(articles).toHaveLength(13)
         articles.forEach((article) => {
-          expect(article).toMatchObject({
-            article_id: expect.any(Number),
-            title: expect.any(String),
-            topic: expect.any(String),
-            author: expect.any(String),
-            created_at: expect.any(String),
-            votes: expect.any(Number),
-            article_img_url: expect.any(String),
-            comment_count: expect.any(Number)
-        });
+          expect(article).not.toHaveProperty("body")
+        })
       })
     })
-  })
 
-  test("200: Article objects do not contain a body property", () => {
-    return request(app)
-    .get("/api/articles")
-    .expect(200)
-    .then(({body: { articles }}) => {
-      expect(articles).toHaveLength(13)
-      articles.forEach((article) => {
-        expect(article).not.toHaveProperty("body")
+    test("200: Response is ordered by date descending order by default", () => {
+      return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({body : {articles }}) => {
+        expect(articles).toBeSortedBy('created_at', { descending: true })
       })
     })
   })
+  describe("?sort_by", () => {
+    test("200: Accepts a sort_by query, which defaults to the created at data", () => {
+        return request(app)
+        .get("/api/articles?sort_by")
```

Comment:
For this test, we should be checking that the endpoint of `/api/articles` defaults toe created_at.